src/Serilog/Serilog.csproj

```diff
@@ -3,8 +3,12 @@
     <Description>Simple .NET logging with fully-structured events</Description>
     <VersionPrefix>4.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
+    <!-- .NET Framework version targeting is frozen at these two TFMs. -->
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462;net471</TargetFrameworks>
-    <TargetFrameworks>$(TargetFrameworks);netstandard2.1;netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
+    <!-- Policy is to trim TFM-specific builds to `netstandard2.0`,
```

Comment:
```suggestion
    <!-- Policy is to trim TFM-specific builds to `netstandard2.0`, `net6.0`,
```
I think `net6.0` should also be on the 'we always compile for it' list - reasoning being that it provides a stable base.

Otherwise, when it gets dropped, you're forced to choose:
- update your minimum to `net8.0` because Serilog forced you to
- make accommodations (add shims) so you can retarget to `netstandard2.0`

Where netstandard2.0 is not feasible, it's also the fixed point of stability that NS2.0 normally is